packages/base-conf/definition.yaml

```diff
@@ -0,0 +1,3 @@
+name: "base-conf"
+category: "system"
+version: "0.0.1+24"
```

Comment:
For convenience while developing I created this config package, it basically holds the layout setup.